### PR TITLE
Elijah

### PR DESCRIPTION
**Pull request by BassettUNC**
Created A reusable nav Bar. It has complete functionality but lacks animations which may be nice in the future. We also need to decide what navitems we want the header to contain.

### CODE REVIEW

File: src/components/TopNavBar/NavContent.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        <div className="navContent">
+            <div id="hamBox">
+                <div id="hamButton">
+                    <Hamburger />
+                </div>
+                <ul id="hamNavList">
+                    <li className="navItem hamNavItem"><a href="../Home" className="navItemLink">placeHolder1</a></li>
+                    <li className="navItem hamNavItem"><a href="../Home" className="navItemLink">placeHolder2</a></li>
+                    <li className="navItem hamNavItem"><a href="../Home" className="navItemLink">placeHolder3</a></li>
+                    <li className="navItem hamNavItem"><a href="../Home" className="navItemLink">placeHolder4</a></li>
```
Comment (rpranith) [inline src/components/TopNavBar/NavContent.js:16]: I think the figma design has the nav links. Here is the link: https://www.figma.com/file/hnVa7Jpi6vMFDIVyfTKyR5/PORCH-Initial-Design?node-id=88%3A2

There is a little tab on the left of the figma that says pages, and the prototype page has the designs for that.

**Review by rpranith — COMMENTED** [review]
Pls see comments

**Comment by BassettUNC** [comment]
Ok I updated the links and made the dropdown better resemble the figma design.

**Comment by rpranith** [comment]
Great job really amazing how fast you picked this up!

**Pull request merged**